src/RenderWebGL.js

```diff
@@ -629,30 +656,46 @@ class RenderWebGL extends EventEmitter {
             gl.stencilFunc(gl.ALWAYS, 1, 1);
             gl.stencilOp(gl.KEEP, gl.KEEP, gl.REPLACE);
             gl.colorMask(false, false, false, false);
-            this._drawThese([drawableID], ShaderManager.DRAW_MODE.silhouette, projection);
+            this._drawThese([drawableID, ...candidateIDs], ShaderManager.DRAW_MODE.silhouette, projection);
 
             gl.stencilFunc(gl.EQUAL, 1, 1);
             gl.stencilOp(gl.KEEP, gl.KEEP, gl.KEEP);
             gl.colorMask(true, true, true, true);
 
-            this._drawThese(candidateIDs, ShaderManager.DRAW_MODE.silhouette, projection,
-                {idFilterFunc: testID => testID !== drawableID}
-            );
+            this._drawThese(candidateIDs.concat([drawableID]), ShaderManager.DRAW_MODE.silhouette, projection);
```

Comment:
remove `concat` here for a truer bench of the "old way" - but this all gets blown away long term